spec/features/admin/admin_edits_an_item_spec.rb

```diff
@@ -0,0 +1,89 @@
+require 'rails_helper'
+
+RSpec.feature do
+  before do
+    @item = create(:item)
+    admin = create(:user, role: 1)
+    login_stub(admin)
+    visit admin_items_path
+    click_on 'Edit'
+  end
+
+  context 'admin' do
+    scenario 'can edit an item\'s title' do
+      expect(current_path).to eq edit_admin_item_path(@item)
+      expect(page).to_not have_content 'NewTitle'
+
+      fill_in 'item[name]', with: 'NewTitle'
+      click_on 'Update'
+
+      expect(page).to have_content 'Item Updated!'
+      expect(page).to have_content 'NewTitle'
+    end
+
+    scenario 'can edit an item\'s description' do
+      expect(page).to_not have_content 'New description'
+
+      fill_in 'item[description]', with: 'New description'
+      click_on 'Update'
+
+      expect(page).to have_content 'Item Updated!'
+      expect(page).to have_content 'New description'
+    end
+
+    scenario 'can edit an item\'s image' do
+      expect(page).to_not have_xpath("//img[@src=\"/uploads/test.jpeg\"]")
```

Comment:
Me too! If you have any tips/tricks/any kind of understanding I'd love to learn more about this